test/TabExpansion.Tests.ps1

```diff
@@ -357,7 +358,7 @@ Describe 'TabExpansion Tests' {
             $result | Should -BeExactly "'v2.0.0'''"
         }
         It 'Tab completes add file in working dir with special char as quoted' {
-            $filename = 'foo{bar} (x86).txt';
+            $filename = 'foo{bar} ��(x86).txt';
```

Comment:
What's with the unrecognized chars?  Is that part of the "special char" test?  Might be worth adding as another test for emoji chars or whatever that is.  IIRC this test was meant to check for tab completion with PowerShell and/or regex special chars/syntax e.g. `{}` and `()` in the completed text.

Reply:
Was just being lazy, but yeah the emoji is to ensure we're setting `core.quotePath` (and it's working as expected). Will break out as a separate test.

Reply:
> > What's with the unrecognized chars? Is that part of the "special char" test? Might be worth adding as another test for emoji/unprintable chars or whatever that is. IIRC this test was meant to check for tab completion with PowerShell and/or regex special chars/syntax e.g. `{}` and `()` in the completed text.
>
> Was just being lazy, but yeah the emoji is to ensure we're setting `core.quotePath` (and it's working as expected). Will break out as a separate test.

Turns out we already had a test for this: https://github.com/dahlbyk/posh-git/blob/a62f90cb1ff1b66bc05da5787eb561bce64e3da6/test/TabExpansion.Tests.ps1#L222-L226

Relocated into the appropriate `Context`.